### ISSUE TITLE
Access key id from other columns

### ISSUE THREAD

**Issue by thinkerytim** [ratiw/vuetable-2#406]
Is it possible to access the key id of the row in the callback? For example one of my callbacks formats a column as an image. But I'd like to link this image to another page keyed by the id of the row. So I'd like to do something like this in the callback:

    ```formatImage(value) {
        return '<a href="/mysite/object/'+ key_id +'"><img src="' + value + '" class="img-thumbnail"></a>';
    },```

**Comment by ratiw** [comment]
@thinkerytim No, you can't because `callback` is primarily used for formatting column value. You should instead use `__slot` instead. It provides scoped data like `rowData` and `rowIndex`, you have to limitation on `__slot`.

**Comment by kwame101** [comment]
I had this same problem, I fixed by using vue router and __slot but you can create a method and call it like you've already done.

Here is my vuetable compoment code: 
```
<div slot="link" slot-scope="props">
      <router-link :to="{ name: '/post/edit/', params: { id: props.rowData.id } }"  >
              {{ props.rowData.title }}
      </router-link>
</div>

```

And here you just include in your other component like:
```
{
 name: '__slot:link',
 title: 'Title'
}
```

@ratiw could you please update the tutorial to use slot-scope instead of scope?
